### DISCUSSION TITLE
海外用戶能否新增其他付款方式

### DISCUSSION THREAD

**Discussion by Arcelibs** [natfrp/wiki#61] [想法]
比如說Paypal之類的，像台灣地區這沒法用支付寶或者微信支付

**Comment by choneas** [comment]
那就别用Sakura frp[doge]

**Reply by Arcelibs** [reply]
手動doge

**Comment by fakeboboliu** [comment]
well，我们在考虑相关的解决方案，但是因为我们的用户群体和服务器主要集中于中国大陆地区，这个问题的优先级可能不会太高。

得益于特色网络条件，如果你居住在中国大陆之外，你到我们主要服务器的网络条件通常不会太好，在这个情况下我个人更推荐使用更注重海外地区的其它服务。

**Reply by Arcelibs** [reply]
了解，謝謝回覆，我都是用日本節點，速度能跑滿
先請重慶的朋友幫忙購買三個月用用，挺方便的

**Comment by XG8425** [comment]
PayPal真的很好用啊 我一个大陆的都喜欢用PayPal
